test/python/fake_agent/fake.py

```diff
@@ -0,0 +1,5 @@
+from datadog_checks.base.checks import AgentCheck
+
+# Fake agent for testing purposes
+class FakeAgent(AgentCheck):
```

Comment:
nit: can we rename it to `FakeCheck`? Technically this is a check

Reply:
:+1:

Reply:
Do I rename the module too?

Reply:
ah yes please